packages/contracts/test/voting/addresslist-voting.ts

```diff
@@ -1200,5 +1220,77 @@ describe('AddresslistVoting', function () {
         expect(await voting.canExecute(id)).to.equal(true);
       });
     });
+
+    describe('An edge case with >0% support and >=0% participation required in early execution mode', async () => {
```

Comment:
I don't get the description well ! in the beforeEach, we got `supportThreshold` and `minParticipation` = 0.

Reply:
Fixed in f16d3e7becb9668bbd24b3ed065cdbee9af2b98f